.github/workflows/bump.yml

```diff
@@ -0,0 +1,84 @@
+name: bump
+
+on:
+  workflow_dispatch:
+    inputs:
+      stream-target-version:
+        description: 'Stream target version for the update'
+        required: false
+      stream-source-version:
+        description: 'Stream source version for the update, required if Stream target version is specified'
+        required: false
+      horizon-target-version:
+        description: 'Horizon target version for the update'
+        required: false
+      horizon-source-version:
+        description: 'Horizon target version for the update, required if Horizon target version is specified'
+        required: false
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  bumper:
+    runs-on: ubuntu-22.04
+
+    steps:
+
+      - name: Get root absolute path
+        run: |
+          cd ..
+          echo "ROOT_PATH=$(pwd)" >> $GITHUB_ENV
+
+      - name: Cloning Playground
+        uses: actions/checkout@v4
+
+      - name: Get repository absolute path
+        run: echo "REPO_PATH=$(pwd)" >> $GITHUB_ENV
+
+      - name: Display values
+        run: |
+          echo "ROOT_PATH: ${{ env.ROOT_PATH }}"
+          echo "REPO_PATH: ${{ env.REPO_PATH }}"
+
+      - name: Cloning Stream
+        if: ${{ inputs.stream-target-version }}
+        run: |
+          cd ${{ env.ROOT_PATH }}
+          git clone -b v${{ inputs.stream-target-version }} "https://milairhu:${{ secrets.API_TOKEN_GITLAB }}@gitlab.com/evertrust/stream.git"
```

Comment:
The entire clone URL should be a variable

Reply:
Fixed c61ac5a43d8e8e66c2118184e411a2707bae2210